app/konnect/service-catalog/index.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Service catalog
+subtitle: Track every service across your architecture
+content-type: explanation
+---
+
+{{site.konnect_saas}}'s Service Catalog offers a comprehensive catalog of all services running in your organization, both {{site.base_gateway}} services and external integration resources.
+
+<!-- vale off-->
+{% mermaid %}
+graph LR
+  %% Define styles for icons and nodes
+  classDef integration fill:#f7f7f7,stroke:#333,stroke-width:2px;
+  classDef resource fill:#e1f7d5,stroke:#333,stroke-width:2px;
+  classDef konnect fill:#d0e5ff,stroke:#333,stroke-width:2px;
+  classDef serviceCatalog fill:#e1f7d5,stroke:#333,stroke-width:2px;
+  classDef billingInventory fill:#f0f0f0,stroke:#333,stroke-width:2px;
+
+  %% Define the resources
+  GitHub[<img src="https://img.icons8.com/ios/50/000000/github.png" width="24" height="24"/> GitHub]:::resource
+  PagerDuty[<img src="https://play-lh.googleusercontent.com/E-zhAf4KJ6JDDXmQfQxBprn2sATGYUMkOEqLQX5HAQQtiwDZJg4c8sQd7deb6nCZCwU=w480-h960-rw" width="24" height="24"/> PagerDuty]:::resource
+  KongGateway[<img src="https://raw.githubusercontent.com/Kong/docs.konghq.com/main/app/assets/images/icons/kong-gradient.svg" width="15" height="15"/> Kong Gateway]:::resource
+
+  %% Define the Konnect Service Catalog and its subgraphs
+  subgraph KonnectServiceCatalog[Konnect Service Catalog]
+    direction TB
+
+    %% Define the Billing Service subgraph inside Konnect Service Catalog
+    subgraph BillingService[Billing Service]
+      direction LR
+      KongGatewayService[Kong Gateway service 1]
+      PagerDutyServiceMapping[PagerDuty service 1]
+      GitHubRepoMapping[GitHub repository 1]
+    end
+
+    %% Define the Inventory Service Catalog Service subgraph inside Konnect Service Catalog
+    subgraph InventoryService[Inventory Service]
+      direction LR
+      SomeOtherService[Kong Gateway service 2]
+      AnotherService[PagerDuty service 2]
+      DifferentRepo[GitHub repository 2]
+    end
+
+  end
+
+  %% Connect the resources to the Konnect Service Catalog
+  GitHub -- "GitHub repositories" --> KonnectServiceCatalog
+  PagerDuty -- "PagerDuty services" --> KonnectServiceCatalog
```

Comment:
We alternatively use "Datadog" and "Datadog Monitor" here instead. I worry that seeing "PagerDuty Service" might throw off readers this early on.

Reply:
> I worry that seeing "PagerDuty Service" might throw off readers this early on.

@miko-bautista Could you explain this a bit? 

Once more integrations/resources are added, I'm not sure that all of them should be in this diagram just because it could become cluttered quickly. But I'm happy to switch PagerDuty out with something else! I think Traceable will be in GA, we could add that instead.

Reply:
I wonder if we shouldn't use specific service names here at all. At some point, we're going to have many other services. It might be better to do something like:

* Gateway Manager services (internal) - you already have this one
* External services

Or, "external service 1", "external service 2", "external service 3". (External integration might be the more accurate term). You could then provide an example of what an external integration could be, which is where you could mention pagerduty and github.